package.json

```diff
@@ -153,10 +153,11 @@
     "mixpanel": "^0.16.0",
     "mixpanel-browser": "^2.41.0",
     "nanoid": "^5.0.7",
-    "next": "canary",
+    "next": "14.3.0-canary.63",
```

Comment:
I think this is breaking the build actually. The problem is, the latest canary requires react 19